hoomd/mpcd/DryingDropletStreamingMethod.cc

```diff
@@ -29,9 +29,15 @@ mpcd::DryingDropletStreamingMethod::DryingDropletStreamingMethod(std::shared_ptr
                                                                  mpcd::detail::boundary bc,
                                                                  Scalar density,
                                                                  unsigned int seed)
-    : mpcd::ConfinedStreamingMethod<mpcd::detail::SphereGeometry>(sysdata, cur_timestep, period, phase, std::shared_ptr<mpcd::detail::SphereGeometry>()),
+    : mpcd::ConfinedStreamingMethod<mpcd::detail::SphereGeometry>(sysdata, cur_timestep, period, phase, std::make_shared<mpcd::detail::SphereGeometry>(R->getValue(cur_timestep), 0.0, bc)),
```

Comment:
I am not sure whether this change was needed or if it's  correct or not but without this change the filler doesn't work, the test code shows segmentation fault.